sdk/platform/posix/CMakeLists.txt

```diff
@@ -11,4 +11,3 @@ add_library(az_posix STATIC)
 target_link_libraries(az_posix PRIVATE az_core)
 
 target_sources(az_posix PRIVATE src/az_posix.c)
-
```

Comment:
This change looks fine since we are making it consistent with the other CMakeLists files.